src/network/socks5.py

```diff
@@ -173,4 +200,5 @@ def state_auth_done(self):
         return True
 
     def resolved(self):
```

Comment:
Resolving is done, process the return value. To use this within PyBitmessage, a callback needs to be implemented which hasn't been done yet.